src/js/control-bar/volume-menu-button.js

```diff
@@ -18,7 +18,17 @@ import VolumeBar from './volume-control/volume-bar.js';
  */
 class VolumeMenuButton extends MenuButton {
 
-  constructor(player, options){
+  constructor(player, options={}){
+    // If the vertical option isn't passed at all, default to true.
+    if (options['vertical'] === undefined) {
```

Comment:
No, I don't think so, I just kept doing it for consistency. Should we start just using dot notation at this point?